src/cmd/troubleshoot/endpoint.go

```diff
@@ -0,0 +1,40 @@
+package troubleshoot
+
+func getOneAgentImageEndpoint(troubleshootCtx *troubleshootContext) string {
+	imageEndpoint := troubleshootCtx.dynakube.ApiUrlHost() + "/linux/oneagent"
+
+	customImage := troubleshootCtx.dynakube.CustomOneAgentImage()
+	version := troubleshootCtx.dynakube.Version()
+
+	if customImage != "" {
+		imageEndpoint = customImage
+	} else if version != "" {
+		imageEndpoint = imageEndpoint + ":" + version
+	}
+
+	logInfof("OneAgent image endpoint '%s'", imageEndpoint)
+	return imageEndpoint
+}
+
+func getOneAgentCodeModulesImageEndpoint(troubleshootCtx *troubleshootContext) string {
+	imageEndpoint := troubleshootCtx.dynakube.CodeModulesImage()
+
+	if imageEndpoint != "" {
+		logInfof("OneAgent codeModules image endpoint '%s'", imageEndpoint)
```

Comment:
should we log this in a getter?

Reply:
the getter is `troubleshootCtx.dynakube.CodeModulesImage()`, this function wraps the logging specifically